docs/kmstool.md

```diff
@@ -244,7 +258,7 @@ for securing and managing production applications.
 The instructions for running the sample are given for both Linux and Windows.
 Windows instructions are preceded with `PowerShell`
 
-Start the enclave:
+In a second (new) terminal session, start the enclave in debug mode and connect:
 ```sh
 nitro-cli run-enclave --eif-path kmstool.eif --memory 512 --cpu-count 2 --debug-mode
```

Comment:
Can you please also update the memory value here to 1024 instead of 512.

Reply:
Same below, for the PowerShell section.

Reply:
Updated!

Reply:
Thanks.